### PR TITLE
chore(connectors): update msteams documantation

### PR DESCRIPTION
**Pull request by Oleksiivanov**
Added `expand get chat by id method`

## Are there related marketing activities

https://github.com/camunda/connectors-bundle/issues/186
https://github.com/camunda/camunda-platform-docs/issues/1596
## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md [hunk 1]
```diff
@@ -95,17 +95,26 @@ For example, if you want to send a message in a Microsoft Teams channel, choose
...
+|     Filter      |                                                              List messages in chat                                                              |    No    |     string      | Sets the date range filter for the lastModifiedDateTime and createdDateTime properties. [Learn more about filtering](https://learn.microsoft.com/en-us/graph/filter-query-parameter) |
+|   Message ID    |                                                               Get message in chat                                                               |   Yes    |     string      |                                                                           Microsoft teams chat message id                                                                            |
+
+##### Expand response
+
+You can expand response for method **Get chat by ID**. For this in **Expand response** property you can :
```
Comment (mbiciin) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:113]: I would suggest "For method  **Get chat by ID**, you can get more information in the response by using the dropdown property  **Expand response**. You can choose one of the following values: "

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md [hunk 2]
```diff
@@ -95,17 +95,26 @@ For example, if you want to send a message in a Microsoft Teams channel, choose
...
+
+- choose **With chat members**, for getting information about members of chat;
```
Comment (mbiciin) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:115]: - **With chat members**, to get information about chat members

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md [hunk 3]
```diff
@@ -95,17 +95,26 @@ For example, if you want to send a message in a Microsoft Teams channel, choose
...
+- choose **With last message preview**, for getting last message in chat. This function doesn't work for [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type.
```
Comment (mbiciin) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:116]: - **With last message preview**, to get last message in chat. This function doesn't work with [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md [hunk 4]
```diff
@@ -95,17 +95,26 @@ For example, if you want to send a message in a Microsoft Teams channel, choose
...
+- choose **Without expand**, for getting main information about chat.
```
Comment (mbiciin) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:117]: - **Without expand**, to get main information about chat

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md (hunk 1 above)
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:113]:
```suggestion
You can expand response for method **Get chat by ID**. For this in the **Expand response** property you can:
```

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md (hunk 2 above)
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:115]:
```suggestion
- select **With chat members**, for getting information about members of chat;
```

File: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md (hunk 3 above)
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md:116]:
```suggestion
- select **With last message preview**, for getting last message in chat. **Note:** this function doesn't work for [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type.
```

**Review by mbiciin — COMMENTED** [review]
In general, looks good to me.

**Review by igpetrov — CHANGES_REQUESTED** [review]
Added comments. Applicable for all files.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**